### PR TITLE
ASR: typo fix from #724

### PR DESCRIPTION
**Pull request by pcuenca**
and minor nits.

### CODE REVIEW

File: packages/tasks/src/tasks/automatic-speech-recognition/about.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ def query(filename):
 data = query("sample1.flac")
 ```
 
-You can also use[huggingface.js](https://github.com/huggingface/huggingface.js), the JavaScript client, to transcribe models with the Inference API.
+You can also use [huggingface.js](https://github.com/huggingface/huggingface.js), the JavaScript client, to transcribe audio with the Serverless Inference API.
```
Comment (Vaibhavs10) [inline packages/tasks/src/tasks/automatic-speech-recognition/about.md:39]: Oh damnn, I'm blind! thanks! 🤗

**Review by Wauplin — APPROVED** [review]

File: packages/tasks/src/tasks/automatic-speech-recognition/about.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ await inference.automaticSpeechRecognition({
 });
 ```
 
-For transformers compatible models like Whisper, Wav2Vec2, HuBERT, etc. You can also run inference in Python using transformers as follows:
+This works for transformers-compatible models like Whisper, Wav2Vec2, HuBERT, etc. You can also run inference in Python using transformers as follows:
```
Comment (Vaibhavs10) [inline packages/tasks/src/tasks/automatic-speech-recognition/about.md:51]: This reads a bit off to me :/

**Review by Vaibhavs10 — APPROVED** [review]
But if you think it reads then feel free to go-ahead and merge.

File: packages/tasks/src/tasks/automatic-speech-recognition/about.md (hunk 2 above)
Comment (osanseviero) [inline packages/tasks/src/tasks/automatic-speech-recognition/about.md:51]: Mmm both are unclear to me, maybe this?

```suggestion
For transformers-compatible models like Whisper, Wav2Vec2, and HuBERT, you can also run inference with the library as follows:
```

**Review by osanseviero — APPROVED** [review]

File: packages/tasks/src/tasks/automatic-speech-recognition/about.md (hunk 2 above)
Comment (pcuenca) [reply packages/tasks/src/tasks/automatic-speech-recognition/about.md:51]: haha, I thought the original intent was to say that the previous api methods work for transformers models. But it looks fine like this imo.

Comment (pcuenca) [reply packages/tasks/src/tasks/automatic-speech-recognition/about.md]: ping @Vaibhavs10 for confirmation

**Review by Vaibhavs10 — APPROVED** [review]
LGTM! Thanks! 🤗

File: packages/tasks/src/tasks/automatic-speech-recognition/about.md (hunk 2 above)
Comment (Vaibhavs10) [reply packages/tasks/src/tasks/automatic-speech-recognition/about.md]: Resolved in the latest commit!

**Pull request merged**